First case.R

```diff
@@ -0,0 +1,92 @@
+library(tidyverse)
+library(lubridate)
+
+#Calculates start date for each country
+
+#col_num <- apply(cases[,5:ncol(cases)], 1, function(x){first_case = min(which(x>0))})
+
+#first_case <- names(cases[,5:ncol(cases)])[col_num]
+
+#cases <- cbind(first_case, cases)
+
+
+# reading in files
+
+cases = read.csv(".\\intermediate_data\\jhu_cases_01.26.2021.csv", check.names = FALSE)
```

Comment:
Not a big deal, but you can also do one forward slash instead of two backslashes.